chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -108,6 +108,9 @@ edp:
   secret: "TBD"
   secretName: "edp-creds"
 
+audit-logs:
+  scriptName: "audit-log-script"
```

Comment:
To be consistent with the rest, shouldn't it be named like "scriptConfigMapName"? Just asking